packages/core/admin/_internal/node/staticFiles.ts

```diff
@@ -17,9 +17,7 @@ const getEntryModule = (ctx: BuildContext): string => {
     .map(({ importName, path }) => `import ${importName} from '${path}';`)
     .join('\n');
 
-  const flags = {
-    contentReleases: process.env.FEATURE_FLAG_CONTENT_RELEASES,
-  };
+  const features = ctx.strapi.config.get('features', {});
```

Comment:
This is cool, lets move so it's added to the buildContext that way we can access it anywhere e.g. if we want to do `unstable_viteBundler` for example 🤓

Reply:
it should be undefined if we don't have any then we can conditionally render it in the static file